documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,256 @@
+
+
+.The VM is posted with its desired `Running` state
+
+{project-short} creates the VM with its desired `Running` state on the target provider, instead of creating the VM and then running it as an additional operation. link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+
+.The `must-gather` logs can now be loaded only by using the CLI
+
+The {product-short} web console can no longer download logs. With this update, you must download `must-gather` logs by using CLI commands. For more information, see link:https://github.com/openshift/must-gather-operator[Must Gather Operator].
+
+
+.{project-short} no longer runs `pvc-init` pods when migrating from vSphere
+
+{project-short} no longer runs `pvc-init` pods during cold migration from a vSphere provider to the {ocp-name} cluster that {project-short} is deployed on. However, in other flows where data volumes are used, they are set with the `cdi.kubevirt.io/storage.bind.immediate.requested` annotation, and CDI runs first-consume pods for storage classes with volume binding mode `WaitForFirstConsumer`.
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This part provides features and enhancements introduced in {project-full} 2.6:
+
+.Validation for RHEL 6 guest operating system is now available when migrating VMs with RHEL 6 guest operating system
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system. Therefore, RHEL 6 is not supported in {ocp-name} Virtualization. With this update, a validation of RHEL 6 guest operating system was added to {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+
+.Automatic retrieval of CA certificates for the provider's URL in the console
+
+The ability to retrieve CA certificates, which was available in previous versions, has been restored. The `vSphere Verify certificate` option is in the `add-provider` dialog. This option was removed in the transition to the {ocp} console and has now been added to the console. This functionality is also available for {a-rhv-short}, {osp}, and {ocp-name} providers now. link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+
+.{project-short} UI plugin now uses only a dynamic plugin for the SDK REST API `fetch` methods
+
+The UI plugin now uses a dynamic plugin for the SDK REST API when using the `fetch` methods. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+
+
+.Plans that failed to migrate VMs can now be edited
+
+With this update, you can edit plans that have failed to migrate any VMs. Some plans fail or are canceled because of incorrect network and storage mappings. You can now edit these plans until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
+
+
+.Validation rules are now available for OVA
+
+The validation service includes default validation rules for virtual machines from the Open Virtual Appliance (OVA).
+
+[id="known-issues-26_{context}"]
```

Comment:
@ahadas - is static ips not preserved also an issue?

FYI @liranr23

Reply:
@anarnold97 I'm not sure
on the one hand, yes, it's something that we don't yet have a solution for
on the other hand, MTV requires mapping a network to another network - I wouldn't expect users to think that static IPs are preserved by default, in that case it's more like a missing feature than a known issue

after writing the above, I wouldn't document it as a known issue because later it would need to be a resolved issue while I'd prefer to document keeping static IPs as a new feature when it's introduced

Reply:
Cool